omod/src/main/webapp/uncaughtException.jsp

```diff
@@ -51,195 +55,207 @@
+        <td><textarea rows="10" cols="80" id="recent_steps"></textarea></td>
     </tr>
 
 </table>
 
 <br/> <br/>
 
 <div>
-<openmrs:message code="uncaughtException.additionalInfo" />
-<ul>
-<li><openmrs:message code="uncaughtException.additionalInfo.msg" /></li>
-<li><openmrs:message code="uncaughtException.additionalInfo.version" /></li>
-<li><openmrs:message code="uncaughtException.additionalInfo.appNameAndVersion" /></li>
-<li><openmrs:message code="uncaughtException.additionalInfo.username" /></li>
-<li><openmrs:message code="uncaughtException.additionalInfo.implementationId" /></li>
-<li><openmrs:message code="uncaughtException.additionalInfo.modules" /></li>
-</ul>
+    <openmrs:message code="uncaughtException.additionalInfo"/>
+    <ul>
+        <li><openmrs:message code="uncaughtException.additionalInfo.msg"/></li>
+        <li><openmrs:message code="uncaughtException.additionalInfo.version"/></li>
+        <li><openmrs:message code="uncaughtException.additionalInfo.appNameAndVersion"/></li>
+        <li><openmrs:message code="uncaughtException.additionalInfo.username"/></li>
+        <li><openmrs:message code="uncaughtException.additionalInfo.implementationId"/></li>
+        <li><openmrs:message code="uncaughtException.additionalInfo.modules"/></li>
+    </ul>
 </div>
 
 <div>
 
-<form action="${reportBugUrl}" target="_blank" method="POST" name="errorSubmitForm">
-	<input type="hidden" name="openmrs_version" value="${openmrs_version}" />
-	<input type="hidden" name="server_info" value="${server_info}" />
-	<input type="hidden" name="username" value="${username}" />
-	<input type="hidden" name="implementationId" value="${implementationId}" />
-	<input type="hidden" name="startedModules" value="${startedModules}" />
-	<input type="hidden" name="errorMessage" id="errorMessageElement" value="${errorMessage}" />
-	<input type="hidden" name="stackTrace" id="stackTraceElement" value="${stackTrace}" />
+    <form action="${reportBugUrl}" target="_blank" method="POST" name="errorSubmitForm">
+        <input type="hidden" name="openmrs_version" value="${openmrs_version}"/>
+        <input type="hidden" name="server_info" value="${server_info}"/>
+        <input type="hidden" name="username" value="${username}"/>
+        <input type="hidden" name="implementationId" value="${implementationId}"/>
+        <input type="hidden" name="startedModules" value="${startedModules}"/>
+        <input type="hidden" name="errorMessage" id="errorMessageElement" value="${errorMessage}"/>
+        <input type="hidden" name="stackTrace" id="stackTraceElement" value="${stackTrace}"/>
 
-    <br/>
+        <br/>
 
-    <input type="button" onclick="javascript:submitErrorReport()" value="Report Problem">
-</form>
+        <input type="button" onclick="javascript:submitErrorReport()" value="Report Problem">
+    </form>
 
 </div>
-	
+
+
+<%
+    } else {
+
+%>
+<h2><openmrs:message code="no.authenticated.user.found"/></h2>
```

Comment:
This error message looks like the action was about looking for an authenticated user. What do you think of using something like `Authentication Required`?

Reply:
@dkayiwa resolved